product_docs/docs/epas/15/database_administration/10_edb_resource_manager/creating_resource_groups.mdx

```diff
@@ -0,0 +1,278 @@
+---
+title: "Creating resource groups"
```

Comment:
Can we have a diff nav title here, as this topic discusses creating, altering, and dropping resource groups?

Reply:
Working with resource groups

Reply:
Will do!